CONTRIBUTING.md

```diff
@@ -46,10 +46,11 @@ We'll now temporarily pivot away from the Event Source YAML file and begin mappi
 	1. This may also require researching SaaS platform event schema documentation to verify the existence of certain Event Types
 	1. If the Event Type exists, map the raw event field name against the associated EMM event attributes
 	1. If the Event Type or an event attribute does not exist for an event type, leave blank
-	1. Once the Event Type has been assessed, export a raw event sample in JSON format locally. To ensure peer review is feasible, each `event_type` should have at least one sample event.  This makes it possible for others to see example raw logs from the SaaS platform and the attributes that are available 
-    1. The event samples should be sanitized to remove any sensitive information.  While sanitizing data, it is important to maintain some contextual consistency with the data that is being sanitized.  For example an email field should have a fake, but semantically valid email address 
-    1. Each event example json file should follow the naming convention of `<category>_<event_type>.json` and be placed in the `event_examples` directory for the product.  Some services may generate an event with outcomes such as `success` or `failure`.  In these cases, it is important to have examples of both outcomes where possible.  The naming convention for these files should be `<category>_<event_type>_<outcome>.json`
-1. Now that all your example log files are collected and sanitized, you will need to go into the event sources YAML file and update links to the event examples for each event type.  This will allow users to see the raw log data that was used to generate the mappings
+	1. Once the Event Type has been assessed, export a raw event sample in JSON format locally. To ensure peer review is feasible, each `event_type` should have at least one sample event.  This makes it possible for others to see example raw logs from the SaaS platform and the attributes that are available
+       - The `create_product` MAKEFILE command will create a directory called `event_examples` in the product directory.  This is where the raw event samples should be stored
+       - The MAKEFILE command also auto-generates empty JSON files for each event type. The JSON files following the naming convention of `<category>_<event_type>.json`
+       - Some services may generate an event with outcomes such as `success` or `failure`.  In these cases, it is important to have examples of both outcomes where possible.  The naming convention for these files should be `<category>_<event_type>_<outcome>.json`
```

Comment:
@dtocco do we need to add instructions for where you would create the failed and success examples should they be needed?

Reply:
Agree ... Will create a new task to update since it will require a slight re-work of the doc